src-tauri/src/browser.rs

```diff
@@ -3,6 +3,8 @@
 
 use std::marker::PhantomData;
 
+use crate::docker::ContainerManager;
+
 use fantoccini::{wd::Capabilities, Client, ClientBuilder, Locator};
 
 use crate::types::Result;
```

Comment:
Ain't your editor doing this kind of stuff automatically?

```suggestion
use fantoccini::{wd::Capabilities, Client, ClientBuilder, Locator};

use crate::docker::ContainerManager;
use crate::types::Result;
```